submissions/Roka20012/js-dom/interactive.js

```diff
@@ -0,0 +1,82 @@
+"use strict";
+
+const nav = document.getElementById("navbar");
+const text = document.getElementById("text");
+const burger = document.getElementsByClassName("burger2")[0];
+const a = nav.querySelectorAll("a");
+
+/**
+ * закриває меню якщо ми натискаємо на пункт меню 
+ * коли ширина екрана менша 900px
+ * author: Rostyk Broslavsky
+ * email: broslavsky.rostislaw@gmail.com
+ */
+function hideBurgerMenu(target) {
+    let computedStyle = getComputedStyle(burger);
+
+    if (computedStyle.left === "15px") {
```

Comment:
We don't want to use computedStyled number in our JS. Cause if minor changes will ocure in our css file, we will have to fix our code. Maybe there is opportunity move this to the some class? something like moved-to-the-left. And than in js compare classes. Actually I see you have already have the class `open` can we use it?

Reply:
Thank you.  I did not notice first